event-handlers/src/main/java/no/sikt/nva/nvi/events/persist/UpsertNviCandidateHandler.java

```diff
@@ -34,19 +35,23 @@ public class UpsertNviCandidateHandler implements RequestHandler<SQSEvent, Void>
     private static final String PERSISTENCE_MESSAGE = "Nvi candidate has been persisted for publication: {}";
     private static final String UPSERT_CANDIDATE_DLQ_QUEUE_URL = "UPSERT_CANDIDATE_DLQ_QUEUE_URL";
     private static final String UPSERT_CANDIDATE_FAILED_MESSAGE = "Failed to upsert candidate for publication: {}";
-    private final CandidateRepository repository;
+    private final CandidateRepository candidateRepository;
+    private final PeriodRepository periodRepository;
```

Comment:
Removed this in previous [PR](https://github.com/BIBSYSDEV/nva-nvi/pull/399), adding back:)